src/index.ts

```diff
@@ -15,35 +15,35 @@ export const defaults = {
   command: 'git checkout -b {branch | shellquote} && git commit --allow-empty -m {commit | shellquote}'
 };
 
-/* eslint-disable @typescript-eslint/no-explicit-any */
-const renderer = (templates:any, name:FormatterName):parseFn => {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const renderer = (templates: any, name: FormatterName): ParseFn => {
```

Comment:
Instead of disabling, could we add a type for the template config? It's just an object with string values (and even the keys are known/static as far as I remember).

Reply:
I agree (also work your other type related comment). Can we maybe address type-related changes into a different pr? I feel this one already got quite big. 

Having a separate pr gives me more room to focus. At the same time, this pr imo cleans up my initial commit quite nicely already :)

Reply:
Yes, certainly. 👍

Reply:
The type-PR can be found at https://github.com/bitcrowd/tickety-tick-formatter/pull/3